vagrant/provisioning/roles/mongodb/templates/mongod.conf

```diff
@@ -0,0 +1,45 @@
+# mongod.conf
+
+# for documentation of all options, see:
+#   http://docs.mongodb.org/manual/reference/configuration-options/
+
+# where to write logging data.
+systemLog:
+  destination: file
+  logAppend: true
+  path: /opt/arkcase/log/mongodb/mongod.log
+
+# Where and how to store data.
+storage:
+  dbPath: /opt/arkcase/data/mongo
+  journal:
+    enabled: true
+#  engine:
+#  mmapv1:
+#  wiredTiger:
+
+# how the process runs
+processManagement:
+  fork: true  # fork and run in background
+  pidFilePath: /var/run/mongodb/mongod.pid  # location of pidfile
+  timeZoneInfo: /usr/share/zoneinfo
+
+# network interfaces
+net:
+  port: 27017
+  bindIp: 127.0.0.1  # Enter 0.0.0.0,:: to bind to all IPv4 and IPv6 addresses or, alternatively, use the net.bindIpAll setting.
```

Comment:
You should configure a Public bindIP.
Replace 127.0.0.1 with host address (172.28.123.3)

Reply:
For other services (like mariadb and others) we use 0.0.0.0 to bind to all IPv4 and IPv6 addresses